### PR TITLE
Add wallet import

### PR DESCRIPTION
**Pull request by 0xdieman**
Added a the wallet import functionnality

### CODE REVIEW

File: app/src/main/res/drawable/ic_baseline_login_24.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24"
+    android:tint="?attr/colorControlNormal">
+  <path
+      android:fillColor="@android:color/white"
+      android:pathData="M11,7L9.6,8.4l2.6,2.6H2v2h10.2l-2.6,2.6L11,17l5,-5L11,7zM20,19h-8v2h8c1.1,0 2,-0.9 2,-2V5c0,-1.1 -0.9,-2 -2,-2h-8v2h8V19z"/>
```
Comment (ahominal) [inline app/src/main/res/drawable/ic_baseline_login_24.xml:9]: maybe create a special value for it? seems a bit long

**Review by ahominal — APPROVED** [review]
Overall it's very good ! didn' sport anything suspicious

**Review by vfaltings — APPROVED** [review]
Looks good to me

File: app/src/main/res/drawable/ic_baseline_login_24.xml (hunk 1 above)
Comment (0xdieman) [reply app/src/main/res/drawable/ic_baseline_login_24.xml:9]: That's an xml path, it is supposed to be like that it should be fine

**Pull request merged**